public/app/features/auth-config/AuthConfigPage.tsx

```diff
@@ -79,50 +82,32 @@ export const AuthConfigPageUnconnected = ({ providerStatuses, isLoading, loadSet
     <Page navId="authentication" subTitle={subTitle}>
       <Page.Contents isLoading={isLoading}>
         <h3 className={styles.sectionHeader}>Configured authentication</h3>
-        {!!enabledProviders?.length && (
-          <div className={styles.cardsContainer}>
-            {enabledProviders.map((provider) => (
+        {authProviders.map((provider) =>
+          alreadyConfiguredProviders.includes(provider) ? (
+            <div className={styles.cardsContainer} key={provider.id}>
               <ProviderCard
-                key={provider.id}
                 providerId={provider.id}
                 displayName={provider.displayName}
                 authType={provider.type}
-                enabled={providerStatuses[provider.id]?.enabled}
-                configFoundInIniFile={providerStatuses[provider.id]?.configFoundInIniFile}
+                enabled={providerStatuses[provider.id]?.enabled || providerStatuses[provider.id]?.enabledInUI}
```

Comment:
I don't think that's correct, if `enabledInUI` is `false`, then it should be disabled, shouldn't it?

Reply:
```typescript
                enabled={providerStatuses[provider.id]?.enabled || providerStatuses[provider.id]?.enabledInUI}
```
if either of them are true it is enabled

Reply:
since it can be enabled from the inifile and therefore it is then enabled. (confusing as f i know)

Reply:
this is not true, it is easier than that.

| enabledUI | enabledIni | res |
|---|---|---|
| TRUE | FALSE | TRUE |
| TRUE | TRUE | TRUE |
| FALSE | TRUE | TRUE |
| FALSE | FALSE | FALSE |